x/feemarket/ante/fee.go

```diff
@@ -54,6 +54,9 @@ func (dfd FeeMarketCheckDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simula
 	gas := feeTx.GetGas() // use provided gas limit
 
 	if len(feeCoins) != 1 {
+		if len(feeCoins) == 0 {
```

Comment:
nit: can we not name this `ErrTest`? Seems like it shld be `ErrNoFee` or smth?

Reply:
OOPS this was a test debug that made it in here